Makefile

```diff
@@ -21,13 +24,22 @@ docs: ## generate Sphinx HTML documentation, including API docs
 	$(IN_VENV) $(MAKE) -C doc clean
 	$(IN_VENV) $(MAKE) -C doc html
 
+docs-schema-ready: ## Build Github-flavored Markdown from Galaxy Tool XSD
```

Comment:
You're probably right, since this is a phony target it does not make much sense to depend on these files. I usually like more targets which are files to produce (e.g. `schema.md` in this case), but it's probably not worth the effort.